routes/api/v1/schools.js

```diff
@@ -7,8 +7,8 @@ router.post('/', auth.optional, schools_controller.createSchool);
 
 router.get('/', auth.optional, schools_controller.getAllSchoolsWithPagination);
 
-router.get('/:id', auth.optional, schools_controller.getSchoolById);
+router.get('/search/:term?', auth.optional, schools_controller.searchSchool);
```

Comment:
This is an optional route param.

Reply:
@stormwild 
Search functionality isn't working with http get method. 
However, the same's working with post method

Reply:
@stormwild